specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -5952,6 +5952,15 @@
       },
       "description": "Describes a virtual machine scale set sku."
     },
+    "AutoOSUpgradePolicy": {
+      "properties": {
+        "disableAutoRollback": {
+          "type": "boolean",
```

Comment:
In general the ARM team discourages the use of booleans. Is this API already deployed/finished?

Reply:
@marstr service side code changes are done. Not sure if that is what you asked. There isn't anything pending from our side.

Reply:
Assuming that this already executing on the server, and that it would be a breaking change to fix now:

Given that this is a new linter error, we're going to also need to add a suppression. Before that happens, we'll  will need to be acknowledged by @ravbhatnagar.